pinot-core/src/main/java/org/apache/pinot/core/data/manager/realtime/HLRealtimeSegmentDataManager.java

```diff
@@ -165,7 +165,7 @@ public HLRealtimeSegmentDataManager(final SegmentZKMetadata segmentZKMetadata, f
         _invertedIndexColumns);
 
     _segmentEndTimeThreshold = _start + _streamConfig.getFlushThresholdTimeMillis();
-    _resourceTmpDir = new File(resourceDataDir, "_tmp");
+    _resourceTmpDir = new File(resourceDataDir, "tmp");
```

Comment:
there wasn't really a good wait to access the TableDataManager from the SegmentDataManager, so this is hardcoded for now